Tests/test_Seq_objs.py

```diff
@@ -789,6 +789,60 @@ def test_join(self):
         self.assertRaises(ValueError, spacer.join, ["ATG", "ATG", 5, "ATG"])
         self.assertEqual(NotImplemented, spacer.join(SeqIO.parse(file, 'fasta')))
 
+    def test_join_UnknownSeq(self):
+        """Checks if Seq join correctly concatinates sequence with the spacer."""
+        # Only expect it to take Seq objects and/or strings in an iterable!
+
+        file = 'Fasta/f003'
```

Comment:
Avoid ``file`` as a variable name - it is the name of a built in function. How about ``filename`` here?